egs/wsj/s5/steps/nnet3/components.py

```diff
@@ -26,13 +26,16 @@ def GetSumDescriptor(inputs):
     return sum_descriptors
 
 # adds the input nodes and returns the descriptor
-def AddInputLayer(config_lines, feat_dim, splice_indexes=[0], ivector_dim=0):
+def AddInputLayer(config_lines, feat_dim, splice_indexes=[0], ivector_dim=0, offset_dim=0):
```

Comment:
you can revert these changes to the component-generating code. [anyway all this code is deprecated.]